Report/Chapters/Process.tex

```diff
@@ -20,7 +20,7 @@ \subsection{Process tasks}
 
 \subsection{Implementation and design tasks}
 
-For the very first part of the project, Martijn, Kristel and Derk were assigned to start working on the program's front end design. Wander and Jan-Jelle were to tackle the beginnings of a usable back end, that is, getting the program designed in the front end to run. This split kept existing during most of the project.
+For the very first part of the project, Martijn, Kristel and Derk were assigned to start working on the program's front end design. Wander and Jan-Jelle were to tackle the beginnings of a usable back end, that is, getting the program designed in the front end to run. This split kept existing during most of the project because of the familiarity with the code.
```

Comment:
I know, but I realy don't know how to reword this to reflect that. I think this is an understandable explanation.